src/corePlugins.js

```diff
@@ -1523,6 +1523,14 @@ export let corePlugins = {
     })
   },
 
+  textWrap: ({ addUtilities }) => {
+    addUtilities({
+      '.text-wrap': { 'text-wrap': 'wrap' },
+      '.text-nowrap': { 'text-wrap': 'nowrap' },
+      '.text-balance': { 'text-wrap': 'wrap' },
```

Comment:
```suggestion
      '.text-balance': { 'text-wrap': 'balance' },
```

Reply:
I HAVE NO IDEA WHAT YOU'RE TALKING ABOUT! 🙈